<launch>
  <!-- Resets a specified robot (model_name) to it's spawn position
       being published on the [model_name]/spawn topic. Use this instead of
       relaunching gazebo every time you want to start over. It's faster. -->

  <!-- Name of the gazebo model being reset -->
  <arg name="model_type" default="pioneer" />
  <!-- Topic to find pose to reset to -->

  <node name="reset_node" pkg="uml_3d_race" type="reset_robot" output="screen">
    <param name="model_name" value="$(arg model_name)" />
  </node>

</launch>
